react-events/my-app/src/CustomButton.jsx

```diff
@@ -0,0 +1,17 @@
+function CustomButton({ text, color }) {
+  function onCustomClick() {
```

Comment:
The hope was that you would put this function inside App instead of here in CustomButton.